app/build.gradle

```diff
@@ -64,5 +64,6 @@ dependencies {
     provided 'javax.annotation:jsr250-api:1.0'
     apt 'com.github.lukaspili.autodagger2:autodagger2-compiler:1.1'
     compile 'com.github.lukaspili.autodagger2:autodagger2:1.1'
+    compile 'com.viewpagerindicator:library:2.4.1'
```

Comment:
Small suggestion. There is no need to add additional repository 
```gradle
maven { url "http://dl.bintray.com/populov/maven" }
```
Viewpagerindicator can be fetched directly from `mavenCentral` http://search.maven.org/#artifactdetails%7Cfr.avianey.com.viewpagerindicator%7Clibrary%7C2.4.1.1%7Caar with:
```gradle
compile 'fr.avianey.com.viewpagerindicator:library:2.4.1.1@aar'
```